### PR TITLE
Fix/multiple account dialogs

### PR DESCRIPTION
**Pull request by Tomekkk**

### CODE REVIEW

File: app/src/main/java/com/macoscope/ketchuplunch/presenter/LoginPresenter.kt [hunk 1]
```diff
@@ -20,9 +20,9 @@ class LoginPresenter(val loginView: LoginView,
                      val googleCredentialsWrapper: GoogleCredentialWrapper) {
 
 
-    private val REQUEST_ACCOUNT_PICKER = 1000
-    private val REQUEST_AUTHORIZATION = 1001
-    private val REQUEST_GOOGLE_PLAY_SERVICES = 1002
+    private var REQUEST_ACCOUNT_PICKER = 1000
+    private var REQUEST_AUTHORIZATION = 1001
+    private var REQUEST_GOOGLE_PLAY_SERVICES = 1002
```
Comment (dbacinski) [inline app/src/main/java/com/macoscope/ketchuplunch/presenter/LoginPresenter.kt]: why var? do you want to change those values?

File: app/src/main/java/com/macoscope/ketchuplunch/presenter/LoginPresenter.kt [hunk 2]
```diff
@@ -49,8 +48,11 @@ class LoginPresenter(val loginView: LoginView,
     }
 
     private fun chooseAccount() {
-
-        loginView.chooseAccount(googleCredentialsWrapper.userCredential, REQUEST_ACCOUNT_PICKER)
+        if (loginView.hasAccountPermissions()) {
+            permissionGranted()
```
Comment (dbacinski) [inline app/src/main/java/com/macoscope/ketchuplunch/presenter/LoginPresenter.kt]: openSelectAccountDialog(userCredential, requestCode) call is missing here?

**Comment by dbacinski** [comment]
-1

File: app/src/main/java/com/macoscope/ketchuplunch/presenter/LoginPresenter.kt (hunk 1 above)
Comment (pziomacki) [inline app/src/main/java/com/macoscope/ketchuplunch/presenter/LoginPresenter.kt]: Agree, those variables should stay read-only, hence `val` keyword should be used

**Comment by pziomacki** [comment]
-1

File: app/src/main/java/com/macoscope/ketchuplunch/presenter/LoginPresenter.kt (hunk 2 above)
Comment (Tomekkk) [inline app/src/main/java/com/macoscope/ketchuplunch/presenter/LoginPresenter.kt]: this cause redundant dialog

**Comment by dbacinski** [comment]
+2

**Pull request merged**